src/components/CCPForm/CCPFormPage.tsx

```diff
@@ -0,0 +1,190 @@
+import { useMutation } from '@apollo/react-hooks';
+import { useQuery } from 'react-apollo';
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import '../styles/EventCreationPage.css';
```

Comment:
this prevents the app from compiling/building - is there a reason this is even being imported here? (this css file for `EventCreationPage` will eventually be phased out as well)

Reply:
Sorry, it was remnants of taking import statements from a different file.